util/StringUtils.java

```diff
@@ -7,7 +7,6 @@
 public class StringUtils {
     private StringUtils() {}
```

Comment:
Please format the code appropriately having spaces, or use the reformat tool from IDE.